src/routes/v1/user.route.js

```diff
@@ -8,10 +8,19 @@ import superAdminCheck from '../../middleware/permission.middleware';
 
 const router = Router();
 const validateRequest = validationMiddleware();
+const { resetPassword, reset } = UserController;
 
 router.post('/signup', validateRequest, UserController.createAccount);
 
 router.post('/login', validateRequest, UserController.loginUser);
+router.post('/reset-password', resetPassword);
+router.patch(
```

Comment:
Welldone @olorunwalawrence , this route already exist on this module as it was implemented by @timi-codes , Please remove this

Reply:
I can see that @olorunwalawrence has just added the routes for the reset password feature he implemented